aten/src/ATen/native/native_functions.yaml

```diff
@@ -4139,13 +4139,13 @@
     CPU: _int_mm_out_cpu
     CUDA: _int_mm_out_cuda
 
-- func: _convert_weight_to_int4pack(Tensor self, int innerKTiles) -> Tensor
+- func: _convert_weight_to_int4pack(Tensor self, int innerKTiles, int qGroupSize, int N, Tensor qScaleAndZeros, Tensor bias) -> Tensor
```

Comment:
Can you please document what those extra arguments are doing, specifically what is `N` parameter, and aren't `qGroupSize` can be already informed from the shape of `self`